tests/std/tests/P1502R1_standard_library_header_units/custom_format.py

```diff
@@ -1,129 +1,221 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+import json
 import os
+import re
 
 from stl.test.format import STLTestFormat, TestStep
 from stl.test.tests import TestType
 
 
+# Print noisy progress messages that are useful when working on this script.
+noisyProgress = False
+
+

```

Comment:
Although I haven't auto-formatted this, I observe that auto-formatters (e.g. Prettier) prefer to add such double newlines to Python. I suspect that because its scoping is controlled by indentation instead of braces, having extra newlines between functions makes it clearer where their boundaries are.

Perhaps we should simply choose an auto-formatter (e.g. Prettier or Black) and auto-format this file and/or everything else?

Reply:
I suppose we don't _quite_ have enough Python in the repo to bother developing a standard Python style. This is at best a minor irritation given how little time we send reviewing Python changes. I certainly don't think it rises to the level of someone spending time setting up auto-formatting and checkin validation.

I will let this go.

Reply:
I'll investigate repo-wide auto-formatting later; for now I'd like to keep the double newlines here.